### PR TITLE
Set propel log level

### PR DESCRIPTION
**Pull request by n10ty**

### CODE REVIEW

File: cmd/pricer/main.go [hunk 1]
```diff
@@ -17,10 +17,10 @@ import (
 
 	"github.com/mysteriumnetwork/discovery/config"
 	_ "github.com/mysteriumnetwork/discovery/docs"
+	"github.com/mysteriumnetwork/discovery/middleware"
 	"github.com/mysteriumnetwork/discovery/price"
 	"github.com/mysteriumnetwork/discovery/price/pricing"
 	"github.com/mysteriumnetwork/discovery/price/pricingbyservice"
-	"github.com/mysteriumnetwork/discovery/token"
```
Comment (Guillembonet) [inline cmd/pricer/main.go:23]: I see this token is not modified, isn't it unused now and can be deleted?

**Review by Guillembonet — APPROVED** [review]

File: price/pricing/market.go [hunk 2]
```diff
@@ -65,7 +65,7 @@ func (m *Market) fetchPricing() (float64, error) {
 	for _, v := range m.apis {
 		resp, err := v.GetRateCacheWithFallback([]exchange.Coin{exchange.CoinMYST}, []exchange.Currency{exchange.CurrencyUSD})
 		if err != nil {
-			log.Warn().Err(err).Msg("could not load pricing info")
+			log.Error().Err(err).Msg("could not load pricing info")
```
[the remark this answers is not in the document]
Comment (Guillembonet) [reply price/pricing/market.go:68]: I see it has continue and if it runs out of APIs it returns error. I'm not very familiar with the code so maybe its fine with error level.

[the remark this answers is not in the document]
Comment (n10ty) [reply price/pricing/market.go:68]: Correct, but this is an error. So I think it's better log it as error. No?

File: cmd/pricer/main.go (hunk 1 above)
Comment (n10ty) [reply cmd/pricer/main.go:23]: True. Removed

**Pull request merged**